Doc/library/functions.rst

```diff
@@ -290,6 +296,10 @@ are always available.  They are listed here in alphabetical order.
       Previously, :exc:`TypeError` was raised when null bytes were encountered
       in *source*.
 
+   .. versionadded:: 3.8
+      *flags* can now be used to accept source that would contain a top-level
```

Comment:
``ast.PyCF_ALLOW_TOP_LEVEL_AWAIT`` can now be passed in *flags* to enable support
for top-level await, async for, and async with.

Reply:
Done.